localstack/services/awslambda/provider.py

```diff
@@ -326,6 +361,10 @@ def _create_version_model(
                 id=new_id,
             )
             function.versions[next_version] = new_version
+            if "$LATEST" in function.permissions:
+                function.permissions[
+                    "$LATEST"
+                ].revision_id = FunctionResourcePolicy.new_revision_id()
```

Comment:
nit: Would be great if you could add a 1-2 line comment here explaining the context of this change.

Reply:
sure, I added a clarifying comment